src/v/cloud_topics/manager/manager.cc

```diff
@@ -37,30 +39,31 @@ seastar::future<> cloud_topics_manager::start() {
       model::l1_metastore_topic,
       model::partition_id(0));
 
-    manage_notifications_ = partition_manager_->register_manage_notification(
-      manager_ntp.ns,
-      manager_ntp.tp.topic,
-      [this, manager_ntp](const auto& partition) {
-          if (partition->ntp() == manager_ntp) {
-              start_managing(*partition);
-          }
-      });
+    manage_notifications_
+      = partition_manager_->local().register_manage_notification(
+        manager_ntp.ns,
+        manager_ntp.tp.topic,
+        [this, manager_ntp](const auto& partition) {
+            if (partition->ntp() == manager_ntp) {
+                start_managing(*partition);
+            }
+        });
```

Comment:
these hunks are from reformatting following the addition of `local()` to dereference the sharded service.